server/routes/v1/version.ts

```diff
@@ -99,3 +99,37 @@ export const resetVersionApprovals = [
     return res.json(version)
   },
 ]
+
+export const updateLastViewed = [
+  ensureUserRole('user'),
+  async (req: Request, res: Response) => {
+    const { id, role } = req.params
+    const { user } = req
+    const version = await findVersionById(req.user!, id, { populate: true })
```

Comment:
Null-assertion being used here `req.user!`, our eslint rules forbid this. If user is optional we should handle it by using some sort of conditional logic